pkg/descheduler/strategies/lownodeutilization.go

```diff
@@ -301,7 +320,10 @@ func evictPodsFromTargetNodes(
 	for _, node := range lowNodes {
 		taintsOfLowNodes[node.node.Name] = node.node.Spec.Taints
 
-		for name := range totalAvailableUsage {
+		for name := range node.highResourceThreshold {
```

Comment:
Similar here with `getResourceNames`. You can construct the list of all relevant resources and pass it into `evictPodsFromTargetNodes` and other functions where it makes sense.

Reply:
s/highThreshold/resourceNames for the reason mentioned above